Feature/7515 Added zip manager and fixed some breaking code

firedeathz opened the pull request:
followanalytics/fa-issues#7515

- Added a script that compresses the necessary files of the template to form the final uploadable zip file and gives it an appropriate name, according to the template version defined on the `definition.json` file
- Fixed some templates that were breaking when receiving them as inapps on our campaigns
- CSS fix for one image popup template on landscape mode
- some `definition.json` fixes/changes for some templates

cjosepha commented:
When does this will finally be available ton prod?

jjoaquim commented:
@cjosepha as soon as this is merged you can test the templates using one of the test apps.

firedeathz commented on multi-screen/css/style.scss:
```diff
@@ -16,39 +16,37 @@ body#popupTemplate.backdrop {
 
     &--current {
-      transform: translateX(0);
+      transform: translateY(-50%);
```
Why did it change from the X axis to the Y axis? Do the pages scroll vertically now somehow?

Ksm125 replied:
the `absolute` property does not default centered verticaly on old OS version, `Tested on iOS < 9 ` . So i had to add `top: 50%` and then `translateY(-50%)` to get it centered.

Ksm125 replied:
didn't find other way to make it center properly

firedeathz commented on multi-screen/css/style.scss:
```diff
@@ -16,39 +16,37 @@ body#popupTemplate.backdrop {
 
     &--previous {
-      transform: translateX(calc(-100% - 2em));
+      transform: translate(calc(-100% - 2em), -50%);
```
Based on the comment above, not sure if this makes sense yet.

Ksm125 replied:
same as above

firedeathz committed:
followanalytics/fa-issues#7515 Added zip manager js to compress each template and give it the according name

- CSS fix for one image popup template on landscape mode
- some definition.json fixes / changes
